_docs/releasing_version.md

```diff
@@ -20,13 +32,15 @@ ruby 1.8 support has been dropped.)
 If necessary, run `bundle install` before continuing. If you want you can also only install the needed gems:
 
 ```bash
-bundle install --path .vendor/ --without system_tests development
+# You can add --jobs with the number of cores you have to run this much faster in parallel.
+bundle install --path .vendor --jobs 8
```

Comment:
`nproc` helps with this on machines with coreutils (Linux):

```suggestion
bundle install --path .vendor --jobs $(nproc)
```

Reply:
The `--path` option in bundler is deprecated. Instead, it's recommend to use `bundle config` to persistently set it.

Reply:
Also, setting the path to something like `~/.vendor` avoid polluting the "repository" and also allows to only download once the dependencies when working with multiple modules.

Reply:
So for me that's the default, so I never understood why people want to use `.vendor`. I have this snippet in my `~/.bashrc`:
```sh
# https://github.com/rubygems/rubygems/issues/4031
# https://bugzilla.redhat.com/show_bug.cgi?id=1574594
if command -v ruby > /dev/null ; then
        GEM_HOME="$(ruby -e 'print Gem.user_dir')"
        export GEM_HOME
fi
```

Nowadays that resolves to `~/.gem/ruby`. I will note that I have to unset it when I use `rbenv` because it messes up, but I never set the bundler path. IMHO the biggest benefit is that you do get the isolation from bundler, but can reuse gems between different modules.

Reply:
That. Looks. Awesome :heart_eyes: !  That resolves in fact to `~/.gem/ruby/<RUBY VERSION>` which means that when your version of Ruby change (on rolling releases OS), you don't have to choose between fighting with gems that needs to be rebuilt or removing the whole bundler directory, because.  This add one more indirection level that might be confusing for beginners, but that looks so much better (I was not aware of this and tuned my config accordingly and I am very happy) :100: !

Reply:
Yes, those are great and in https://github.com/rubygems/rubygems/discussions/5878 there are many tips on how to make it default. And there's a general consensus that it should be default, but the concrete implementation has yet to surface (IIRC there is a PR in progress).